openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -180,7 +180,7 @@ def _log_and_raise_inactive_user_auth_error(unauthenticated_user):
         error_code='inactive-user',
         context={
             'platformName': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
-            'supportLink': configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK)
+            'supportLink': configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK) or ''
```

Comment:
Shouldn’t it fallback to `settings.SUPPORT_SITE_LINK` if it's not set in the site configuration which actually is an empty string?

Reply:
currently on my local devstack both these values are not set
- My site config do not have anything
- SUPPORT_SITE_LINK = ''

but when I debug and run this statement as whole it returns {}
```shell
> configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK)
{}
```

Reply:
That is I guess a bug in this code https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/site_configuration/helpers.py#L118-L129 which is expecting to raise an exception upon updating a non-dict object.

Reply:
Will look into it and update the change

Reply:
Thank you! 

One thing to notice here is that this only happens if the default value is an empty string instead of `None`. If the configuration is not defined and the default is a `None` then it works as expected.